test/structure/structure_region/test_capping.py

```diff
@@ -13,13 +14,195 @@
 import enzy_htp.structure.structure_region.capping as capping
 import enzy_htp.structure.structure_region as stru_regi
 import enzy_htp.structure.structure_selection as stru_sele
+from enzy_htp.structure.structure_region.residue_caps import SUPPORTED_CAPS, ResidueCap, CH3Cap 
 
 CURR_FILE = os.path.abspath(__file__)
 CURR_DIR = os.path.dirname(CURR_FILE)
 DATA_DIR = f"{CURR_DIR}/../data/"
 WORK_DIR = f"{CURR_DIR}/../work_dir/"
 sp = PDBParser()
 
+def test_capping_with_residue_terminals_tri_a(helpers):
+    """as name. use an example region from selection.
+    Just make sure not execptions is raised for now.
+    TODO complete this test"""
+    test_stru = sp.get_structure(f"{DATA_DIR}tri_alanine.pdb")
+    sele = stru_sele.select_stru(
+        test_stru, "resi 2")
+    test_region = stru_regi.StructureRegion(atoms=sele.atoms)
+    unique_names = set()
+    for atom in test_region.atoms:
+        unique_names.add(atom.name)
+
+    assert len(test_region.atoms) == len(unique_names)
+        
+
+def test_all_cap_types_supported():
```

Comment:
Please have the function name in these titles. I was not sure what this test is for until I read the code.

Reply:
I tried to make them more descriptive. Hopefully these are good?

Reply:
I hope we can have a function name that is the target for this test noted either in the test function name or at least in docstring